src/main/webapp/postcss.config.js

```diff
@@ -1,21 +1,8 @@
-const formatAntStyles = require("./styles");
-
 /**
  * Used by Post CSS through the WebPack build process
  */
 
-/*
-Import custom styles for Ant Design
- */
-const modifyVars = formatAntStyles();
-
 module.exports = {
-  options: {
```

Comment:
These options are set in webpack.config.js so I figured they were unnecessary here.